tests/app/celery/test_lookup_recipient_communication_permissions_task.py

```diff
@@ -174,12 +174,12 @@ def test_recipient_has_given_permission_with_default_send_indicator_and_no_prefe
         new=mocked_va_profile_client
     )
 
-    mock_communication_item = mocker.Mock(CommunicationItem)
-    mock_communication_item.default_send_indicator = send_indicator
+    test_communication_item = CommunicationItem
+    test_communication_item.default_send_indicator = send_indicator
```

Comment:
This doesn't seem to fail the test, but it's passing by accident.  "CommunicationItem" is the class, not an instance, and you've made "default_send_indicator" a static attribute.  You should instead instantiate:
```
test_communication_item = CommunicationItem(id=uuid4(), va_profile_item_id=1, name="name", default_send_indicator=send_indicator)
```

```
>>> class Test:
...     pass
... 
>>> Test.static_attr = "hello"

>>> a = Test()
>>> b = Test()

>>> a.static_attr
'hello'
>>> b.static_attr
'hello'

>>> b.static_attr = "change this"
>>> a.static_attr
'hello'
>>> b.static_attr
'change this'

>>> Test.static_attr = "something else"
>>> a.static_attr
'something else'
>>> b.static_attr
'change this'
```